src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -255,19 +249,35 @@ public void burnFlashSparks() {
     armMotor.burnFlash();
   }
 
+  public void setArmPositionGoal(double desiredArmPosDeg) {
+    armController.setGoal(desiredArmPosDeg);
```

Comment:
We need to do something so that we don't call setGoal constantly but only when the goal changes